projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-joltw.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * joltw - To be named. jetpack#19212
+ * Jetpack interface for calling atomic-auth-proxy on wpcom.
+ *
+ * @package automattic/jetpack
+ * @since ???
+ */
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * joltw - Call wpcom atomic-auth-proxy from jetpack.
+ *
+ * @since ???
+ */
+class WPCOM_REST_API_V2_Endpoint_Joltw extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'joltw';
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register the route.
+	 */
+	public function register_routes() {
+		register_rest_route($this->namespace, '/' . $this->rest_base . '/file(?P<path>/.+)?', [
```

Comment:
This would end with `/file` I think, and drop the regex (acceptable arguments, like `path`, would be listed below in `args`.